set.mm

```diff
@@ -477,20 +477,23 @@ The Metamath (Metamath-exe) program Proof Assistant requires proofs to
        _Instructions_:
        <HTML><ol><li>Assign this inference to any unknown step in the proof.
        Typically, the last unknown step is the most convenient, since
-       <code>MM-PA> ASSIGN LAST</code> can be used.  This step will be
+       <code>MM-PA&gt; ASSIGN LAST</code> can be used.  This step will be
        replicated in hypothesis a1ii.1, from where the development of the main
        proof can continue.</li><li>Develop the independent subproof backwards
-       from hypothesis a1ii.2.  If desired, use a <code>MM-PA> LET STEP</code>
+       from hypothesis a1ii.2.  If desired, use a
+       <code>MM-PA&gt; LET STEP</code>
        command to pre-assign the conclusion of the independent subproof to
        a1ii.2.</li><li>After the independent subproof is complete, use
-       <code>MM-PA> IMPROVE ALL</code> to assign it automatically to an unknown
+       <code>MM-PA&gt; IMPROVE ALL</code>
+       to assign it automatically to an unknown
        step in the main proof that matches it.</li><li>After the entire proof
        is complete, use <code>MM-PA> MINIMIZE_WITH *</code> to clean up
        (discard) all ~ a1ii references automatically.</ol></HTML>
 
        This can also be used to apply subproofs from other theorems.  In step
        2, simply assign the theorem to a1ii.2, and run
-       <HTML><code>MM-PA> EXPAND <theorem></code></HTML> to "import" a subproof
+       <HTML><code>MM-PA&gt; EXPAND &lt;theorem&gt;</code></HTML>
+       to "import" a subproof
```

Comment:
+1, I missed that `<theorem>` would be interpreted as such
![image](https://github.com/user-attachments/assets/852c4569-fc4a-4022-83af-2e31ae2a1c24)

This fixes how https://us.metamath.org/mpeuni/a1ii.html states `MMPA> EXPAND` only... which now that I think about it, might be a better rewording to be consistent?